### PR TITLE
Add STT examples for custom grammars

### PR DESCRIPTION
**Pull request by AnthonyOliveri**

### CODE REVIEW

File: SpeechToTextV1.playground/Pages/Custom grammars.xcplaygroundpage/Contents.swift
Comment (mkistler) [inline SpeechToTextV1.playground/Pages/Custom grammars.xcplaygroundpage/Contents.swift]: This title should probably be "Custom grammars".

File: SpeechToTextV1.playground/Pages/Custom grammars.xcplaygroundpage/Contents.swift
Comment (mkistler) [inline SpeechToTextV1.playground/Pages/Custom grammars.xcplaygroundpage/Contents.swift:30]: For these methods that give no response, I have started using the description from the 200 status code as the message text in this print. This is a _very minor_ thing, but gives a nice linkage back to the docs. So in this case I would use
```
print("The request succeeded.")
```
I'll leave it up to you to decide if you want to follow that convention.

**Review by mkistler — APPROVED** [review]
Looks good! I left a couple minor comments which I trust you will handle appropriately.

File: SpeechToTextV1.playground/Pages/Custom grammars.xcplaygroundpage/Contents.swift
Comment (AnthonyOliveri) [reply SpeechToTextV1.playground/Pages/Custom grammars.xcplaygroundpage/Contents.swift:30]: I still don't understand how a message like "The request succeeded." would be better.

Comment (mkistler) [reply SpeechToTextV1.playground/Pages/Custom grammars.xcplaygroundpage/Contents.swift:30]: I don't disagree, but it's more a matter of treating the API doc as the "single source of truth".  If the description in the API doc is bad, we should fix it there.

But we can pursue this on a separate track -- it shouldn't hold up this work.

Comment (AnthonyOliveri) [reply SpeechToTextV1.playground/Pages/Custom grammars.xcplaygroundpage/Contents.swift:30]: Could you show an example of this in the API docs?

**Pull request merged**
